### PR TITLE
[WIP] Developer guide

### PR DESCRIPTION
**Pull request by IrvingHe000**
- [X] Acknowledgements
- [X] Instructions for Manual Testing
- [x] implementations of future features
- [x] Update implementation of addShift

### CODE REVIEW

**Review by Gabau — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -665,6 +704,30 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: A new shift will be added to the first person in the list schedule. In this case, 
+       the date of that shift will be `2021-11-08`, and the slot is `morning`.
+       Otherwise, no shift is added and error details will be shown in the status message.
+       
+    1. Test case: `addShift -i 1 d/Monday-0 da/2021-11-06 da/2021-12-06` 
+       Expected: New shifts will be added to the first person in the list schedule. In this case,
+       the date of shifts will be `2021-11-08`, `2021-11-15`, `2021-11-22`, `2021-11-29`, `2021-12-06`
+       and the slot is `morning`. Otherwise, no shift is added and error details will be shown in the status message.
+       
+    1. Other incorrect delete commands to try: `addShift -i 0 d/Monday-0`, `addShift -i x d/Monday-0`, (where
```
Comment (Tetrerox) [inline docs/DeveloperGuide.md:727]: Need to remove the 'delete' word.

**Review by Tetrerox — CHANGES_REQUESTED** [review]
LGTM except for this one small issue.

**Review by Tetrerox — APPROVED** [review]
LGTM.

**Pull request merged**